docs/install.md

```diff
@@ -7,31 +7,27 @@ You can install Calypso directly on your machine by following the next steps, or
 1.	Check that you have all prerequisites (Git, Node, NPM). See [below](install.md#prerequisites) for more details.
 2.	Clone this repository locally.
 3.	Add `127.0.0.1 calypso.localhost` to your local `hosts` file.
-4.	Run `npm install`
-5.	Execute `make run` or `make dashboard` (for a more visually-oriented interface) from the root directory of the repository.
-6.	Open [`calypso.localhost:3000`](http://calypso.localhost:3000/) in your browser.
+4.	Execute `npm start` or `npm run dashboard` (for a more visually-oriented interface) from the root directory of the repository.
+5.	Open [`calypso.localhost:3000`](http://calypso.localhost:3000/) in your browser.
 
 ## Prerequisites
 
 **On Mac OS X or Linux**, to be able to clone the repository and run the application you need:
 
 -	Install the [Node.js](http://nodejs.org/) and matching [NPM](https://www.npmjs.com/) [versions](https://nodejs.org/en/download/releases/) listed in the engines section of [package.json](https://github.com/Automattic/wp-calypso/blob/master/package.json). On Mac OS X using [nvm](https://github.com/creationix/nvm) or [n](https://github.com/tj/n) makes it easy to manage `node` versions.
```

Comment:
Changed this (and your other comment) in https://github.com/Automattic/wp-calypso/pull/14631/commits/57139731abe17bb5f79e78ff188f5a2b914aa053